openrpc/README.md

```diff
@@ -13,17 +14,58 @@
   - [Examples (`/openrpc/src/examples/*`)](#examples-openrpcsrcexamples)
   - [Example Pairings (`/openrpc/src/examplePairingObjects/*`)](#example-pairings-openrpcsrcexamplepairingobjects)
 
+## TL;DR
+
+If you're here to add a new method to one of the specfiles, you'll want to:
+
+1. Create a new `<method_name>.json` file in the relevant
+   `/openrpc/src/{stellar-rpc,anchor-platform}/methods` directory. Follow the
+   OpenRPC specification concerning [method objects] for defining this file.
+2. Feel free to use any existing (or create new) `$ref` objects along the way.
+   More about them [later on](#json-refs).
+3. Once your method is defined the way you want it, run the [build](#building)
+   and [validate](#testing) scripts to generate updated "bundled" specfiles.
+4. Create a new Docusaurus page for the method. You can use one of the existing
```

Comment:
Perfect thank you very much! I honestly spent waaay too long to figure out how to do that when I needed to 😓